src/main/java/core/basesyntax/supplier/FigureSupplier.java

```diff
@@ -0,0 +1,54 @@
+package core.basesyntax.supplier;
+
+import core.basesyntax.color.Color;
+import core.basesyntax.figures.Circle;
+import core.basesyntax.figures.Figure;
+import core.basesyntax.figures.FigureName;
+import core.basesyntax.figures.IsoscelesTrapezoid;
+import core.basesyntax.figures.Rectangle;
+import core.basesyntax.figures.RightTriangle;
+import core.basesyntax.figures.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int NUMBER_OF_FIGURE = 5;
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private RandomSupplier randomSupplier = new RandomSupplier();
+
+    public Figure getRandomFigure() {
+        Figure figure;
+        int figureNumber = random.nextInt(NUMBER_OF_FIGURE);
```

Comment:
I agree
`since we have an enum, there is no need to hardcode this number`
see previous comment